website/pages/component/button/example.tsx

```diff
@@ -21,45 +23,58 @@ const knobsConfig: KnobsConfigInterface<{
   size: {
     type: 'select',
     options: Object.values(Size),
-    default: Size.Normal,
+    default: Size.Default,
     label: 'Size',
   },
   disabled: {
     type: 'boolean',
     default: false,
     label: 'Disabled',
   },
+  darkMode: {
+    type: 'boolean',
+    default: false,
+    label: 'Dark Mode',
+  },
   hasHrefProp: {
     type: 'boolean',
     default: false,
     label: 'href',
   },
-  title: {
+  children: {
     type: 'text',
-    default: 'The button title',
-    label: 'Title',
+    default: 'Edit',
+    label: 'Children',
   },
-  glyph: {
+  leftGlyph: {
     type: 'select',
     options: Object.keys(glyphs),
     default: 'Edit',
-    label: 'Glyph',
+    label: 'Left Glyph',
   },
-  children: {
-    type: 'text',
+  rightGlyph: {
+    type: 'select',
+    options: Object.keys(glyphs),
     default: 'Edit',
```

Comment:
Let's default the right glyph to something more realistic, such as CaretDown.

Reply:
👍